.mocky.iOS.yml

```diff
@@ -1,15 +1,14 @@
 sources:
-  - ./SwiftyMocky-Example/Mocky
-  - ./SwiftyMocky-Example/Tests
+  - ./SwiftyMocky-Example/Shared
+  - ./SwiftyMocky-Example/tvOS
```

Comment:
We should scan only sources that belongs to specyfic platform, so in this case we should remove `tvOS` soruces